### PR TITLE
Add KeybindManager.RemoveAll()

### PR DESCRIPTION
**Pull request by gplusplus314**
- [x] This code is up-to-date with the `main` branch.

This PR implements #687.

This PR adds a new method to `(I)KeybindManager` called `RemoveAll()`. As the name implies, it removes all keybinds. See #687 for use-case info.

### CODE REVIEW

File: src/Whim/Keybind/KeybindManager.cs [hunk 1]
```diff
@@ -127,4 +127,13 @@ public bool Remove(string commandId)
...
 		return true;
 	}
+
+	public void RemoveAll()
+	{
+		Logger.Debug("Removing all keybinds");
+		foreach (string commandId in _commandsKeybindsMap.Keys.ToArray())
+		{
+			Remove(commandId);
+		}
```
Comment (dalyIsaac) [inline src/Whim/Keybind/KeybindManager.cs:137]: I this should work?

```suggestion
		_commandsKeybindsMap.Clear();
```

**Review by dalyIsaac — COMMENTED** [review]
If it's alright, could you add a test in `src/Whim.Tests/Keybind/KeybindManagerTests`?

File: src/Whim/Keybind/KeybindManager.cs (hunk 1 above)
Comment (gplusplus314) [reply src/Whim/Keybind/KeybindManager.cs:137]: Maybe... but then what do we do about the equivalent of `_keybindsCommandsMap[keybind].Remove(commandId);` from `Remove()`?

Comment (dalyIsaac) [reply src/Whim/Keybind/KeybindManager.cs:137]: That's true - can you clear both maps?

**Pull request merged**